src/App.tsx

```diff
@@ -2,10 +2,54 @@ import React from 'react'
 import logo from './logo.svg'
 import './App.css'
 import FileImportButton from './Components/FileImportButton'
+import TextField from '@material-ui/core/TextField'
+import xml2js from 'xml2js'
+import Attempt from './types'
 
 const App = () => {
-  function handleFile() {
-    console.log('test')
+  const [attemptHistory, setAttemptHistory] = React.useState<Attempt[]>([])
```

Comment:
Seems odd to store all of this in a single React component.

Reply:
Not super sure how to organize React projects, but most everything so far is just experimentation to figure out how things work.  As I build more parts of the actual project I'll start organizing better probably